### DISCUSSION TITLE
Statistics show that  content is missing

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1689] [General]
![f0320c69f4c9915115dbf181151d94c](https://github.com/user-attachments/assets/99e83ecc-aa62-4db7-88d4-728fdb3b0e00)
[Uploading 西达摩 哈马修 水洗_24-09-08_1911.alog.txt…]()
I don't know why there was a problem.

**Comment by roasterdave** [comment]
The profile did not upload.  Would you try again?

**Comment by MAKOMO** [comment]
Thanks for your report! This should be fixed now.

**Reply by Terracotta-6** [reply]
Have been fixed . thanks a lot !

**Comment by Terracotta-6** [comment]
How to configure?
![59230cc190c1943a7a2d2d69176420d](https://github.com/user-attachments/assets/f107c372-8a6f-4493-b8d1-0c4e313b5dad)
![3be256d9e6c1c9e0d42bfce69fd9571](https://github.com/user-attachments/assets/9149f07c-852f-4a0d-8d5f-b8cd4b128aa7)

**Comment by roasterdave** [comment]
Config>> Statistics>> Statistics tab, uncheck Characteristics to disable data display. 

Right click on the Characteristics data to toggle the data displayed.

**Reply by Terracotta-6** [reply]
Thank you for your answer.
